src/components/diff/NotebookDiff.ts

```diff
@@ -126,19 +145,36 @@ export class NotebookDiff extends Panel implements Git.Diff.IDiffWidget {
     if (this._areUnchangedCellsHidden !== v) {
       Private.toggleShowUnchanged(
         this._scroller,
+        this._isConflict,
         this._areUnchangedCellsHidden
       );
       this._areUnchangedCellsHidden = v;
     }
   }
 
+  /**
+   * Helper to determine if a notebook merge should be shown.
+   */
+  private get _isConflict(): boolean {
+    return !!this._model.base;
```

Comment:
Let's use the model helper (in case in a the future the definition for `hasConflict` changes).

```suggestion
    return this._model.hasConflict;
```